### PR TITLE
Tweaks to command-line help, readme and contribution guidelines

### PR DESCRIPTION
**Pull request by matthewpwilson**
A few suggested tweaks to the command line help, including some examples in the style of core Zowe CLI.

Also I've added a bit of extra info the README just in case anyone comes looking in the near future. We'll need to add more info in here about how to actually use the tool over the next few weeks.

### CODE REVIEW

**Comment by matthewpwilson** [comment]
@markcocker @tasha-m-k any thoughts on these words?

**Review by pcoop — CHANGES_REQUESTED** [review]
Minor comment for Bundle.definition.ts:  "Generate a CICS bundle in the working directory, where the is no package.json...". Presumably the second 'the' should be 'there'. Or perhaps better yet, lose the comma and have "in which there is no package.json".

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
...
+This project aims to create a plugin for Zowe CLI https://github.com/zowe/zowe-cli.  The plugin will provide the capability to deploy applications developed on a workstation to IBM CICS, as CICS bundles. Initially it will support deploying Node.js applications.  It aims to provide an experience similar to deploying to a cloud platform when deploying to CICS.  It will also provide low-level commands for performing individual pieces of the deployment process that could be used as part of a CI/CD pipeline.
 
-## Generation of CICS Bundles
-A CICS Bundle resource defines one or more resources that can be installed into a CICS region. This project is derived from the Zowe CLI Sample Plug-in project, it adds the ability to generate Bundle meta-data in the working directory.
+## Status
+It's currently a work in progress at a fairly early stage. If you'd like to try it out, at the moment you'll need to build the plugin from source as described [here](docs/tutorials/Setup.md). 
+
+## Contributing
+Contributions are welcom, see our [contribution guidelines](CONTRIBUTING.md).
```
Comment (tasha-m-k) [inline README.md]:
```suggestion
Contributions are welcome, see our [contribution guidelines](CONTRIBUTING.md).  
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 # Contribution Guidelines
 The contribution guidelines in this document pertain to contributing to this zowe-cli-cics-deploy-plugin repository.
 
+This project welcomes all contributions. 
+- If you find a bug or would like to suggest an improvement, please raise an issue.
+- If you'd like to contribute a bug fix or small enhancement, please fork the repo and make a pull request.
```
Comment (tasha-m-k) [inline CONTRIBUTING.md]:
```suggestion
- If you would like to contribute a bug fix or small enhancement, please fork the repo and make a pull request.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
...
+- If you'd like to contribute a new feature, please raise an issue describing your proposal so we can discuss it first.
```
Comment (tasha-m-k) [inline CONTRIBUTING.md]:
```suggestion
- If you would like to contribute a new feature, please raise an issue describing your proposal so we can discuss it first.
```

**Review by markcocker — COMMENTED** [review]
Suggest simplifying a bit. Also corrected "welcom"

# zowe-cli-cics-deploy-plugin
This project aims to create a plugin for [Zowe CLI](https://github.com/zowe/zowe-cli) to deploy applications developed on a workstation to IBM CICS Transaction Server for z/OS (CICS). Initially it will support deploying Node.js applications. It aims to provide an experience similar to deploying to a cloud platform when deploying to CICS. It will also provide low-level commands for performing individual steps of the deployment process that could be used as part of a CI/CD pipeline.

## Status
The project is currently a work in progress at a fairly early stage. If you'd like to try it out, you will need to build the plugin from source as described in [setup](docs/tutorials/Setup.md).

## Contributing
Contributions are welcome, see our [contribution guidelines](CONTRIBUTING.md).

File: src/cli/generate/bundle/Bundle.definition.ts [hunk 4]
```diff
@@ -23,11 +23,26 @@ import { PortOption } from "./options/Port.option";
 export const BundleDefinition: ICommandDefinition = {
     name: "bundle",
     aliases: ["b", "bun", "bund"],
-    summary: "Generates a Bundle",
-    description: "CICS Bundle meta-data is generated within the working directory. " +
+    summary: "Generates a bundle",
+    description: "Generate CICS bundle within the working directory. " +
```
Comment (tasha-m-k) [inline src/cli/generate/bundle/Bundle.definition.ts]: Wording is slightly different to whats below
```suggestion
    description: "Generate a CICS bundle in the working directory. " +
```

File: src/cli/generate/bundle/Bundle.definition.ts [hunk 5]
```diff
@@ -23,11 +23,26 @@ import { PortOption } from "./options/Port.option";
 export const BundleDefinition: ICommandDefinition = {
     name: "bundle",
     aliases: ["b", "bun", "bund"],
-    summary: "Generates a Bundle",
-    description: "CICS Bundle meta-data is generated within the working directory. " +
+    summary: "Generates a bundle",
```
Comment (tasha-m-k) [inline src/cli/generate/bundle/Bundle.definition.ts]:
```suggestion
    summary: "Generate a CICS bundle",
```

File: README.md [hunk 6]
```diff
@@ -1,5 +1,9 @@
 # zowe-cli-cics-deploy-plugin
-This repository contains the cics-deploy plug-in for Zowe. Use this project for creating CICS Bundle resources from a Node.js project directory. 
+This project aims to create a plugin for Zowe CLI https://github.com/zowe/zowe-cli.  The plugin will provide the capability to deploy applications developed on a workstation to IBM CICS, as CICS bundles. Initially it will support deploying Node.js applications.  It aims to provide an experience similar to deploying to a cloud platform when deploying to CICS.  It will also provide low-level commands for performing individual pieces of the deployment process that could be used as part of a CI/CD pipeline.
 
-## Generation of CICS Bundles
-A CICS Bundle resource defines one or more resources that can be installed into a CICS region. This project is derived from the Zowe CLI Sample Plug-in project, it adds the ability to generate Bundle meta-data in the working directory.
+## Status
+It's currently a work in progress at a fairly early stage. If you'd like to try it out, at the moment you'll need to build the plugin from source as described [here](docs/tutorials/Setup.md).
```
Comment (tasha-m-k) [inline README.md]:
```suggestion
It is currently a work in progress at a fairly early stage. If you would like to try it out, you need to build the plugin from source as described [here](docs/tutorials/Setup.md). 
```

**Review by tasha-m-k — COMMENTED** [review]
Just a few small wording changes to make it easier to read.

**Review by pcoop — DISMISSED** [review]

**Comment by matthewpwilson** [comment]
I've added some further changes to describe the 'name mangling' behaviour in the help.

**Review by pcoop — APPROVED** [review]

**Pull request merged**